index.js

```diff
@@ -1,30 +1,56 @@
-var parser = require('posthtml-parser');
 var fs = require('fs');
 var path = require('path');
-
+/*
+ * param {Object} options
+ * param {Object} options.root – Root folder path for include
+ * param {Object} options.encoding – File encoding
+ * param {Object} options.addDependencyTo – An object with addDependency() method,
+ *                                          taking file path as an argument.
+ *                                          Called whenever a file is included.
+ * param {Object} options.parse – Parser Object config
+ * param {Object} options.parse.parser – Parser engine
+ * param {Object} options.parse.filter – Filter paths for parser
+ */
 module.exports = function(options) {
     options = options || {};
     options.root = options.root || './';
     options.encoding = options.encoding || 'utf-8';
 
+    var addDependencyTo = options.addDependencyTo || {};
+    var parse = options.parse || {
+        parser: require('posthtml-parser'),
```

Comment:
> version compatibility...

Is this revering to `options.addDependencyTo` here? Deprecate/Remove it please i Update posthtml-loade accordingly adn in time. As long as it work out-of-the-box no one will complain to much about it. e.g postcss-import and potcss.loader is going the same way :)